server/routes/flag.js

```diff
@@ -5,6 +5,21 @@ const Flag = require('../models/flagModel');
 const databaseMaster = require('../DatabaseAccess/databaseMaster');
 const { v4: uuidv4 } = require('uuid');
 
+const PORT = 4001;
+
+const io = require('socket.io')(PORT, {
+    cors: {
+        origin: ['http://localhost:3000'] // client URL
```

Comment:
Might need to change to ['http://localhost:3000', 'https://sentinel-frontend.vercel.app']